app/cmd/cli/app.go

```diff
@@ -9,10 +9,11 @@ import (
 	"strconv"
 	"strings"
 
+	"github.com/spf13/cobra"
+
```

Comment:
Keeping as is. This is actually automatically done by `goimports-reviser` which is a new linter I imported. It separates `pokt-network` impros from other imports.

Reply:
Can you elaborate that confusion a little bit more?  For example, if somebody is confused because they don't understand why there are two public keys in AAT, it's probably because they don't understand the role of those keys.  Saying Application may or may not be the same as Client doesn't explain it.

Reply:
As I mentioned in another comment, we should recomment something in this doc.  In my understanding, application shouldn't be the same as client.  Here's why.

Application private key is very confidential.  Once it's stolen, they can make relays for free.  Relayer application needs to store client key in their application.  This means if they use the same key as app and their application is hacked, their application is hacked.  If they separate application from client, they can unstake app.

Is there any benefit from sharing app and client?